src/App.js

```diff
@@ -2,20 +2,27 @@ import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-
+function useSemiPersistentState () {
+  const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem('savedTodoList')));
+    React.useEffect(() => {
+    localStorage.setItem('savedTodoList', JSON.stringify(todoList));
+    }, [todoList]);
+        return (
```

Comment:
your useState, useEffect, and return are all under the same outer branch of useSemiPersistentState and therefore should have the same indentation